Kitodo/src/main/java/de/sub/goobi/helper/BatchStepHelper.java

```diff
@@ -365,10 +317,15 @@ public String solveProblemForAll() {
         try {
             for (Task s : this.steps) {
                 this.currentStep = s;
-                solveProblem();
+                serviceManager.getWorkflowService().setSolution(getSolution());
+                try {
+                    setCurrentStep(serviceManager.getWorkflowService().solveProblem(this.currentStep, this.solutionTask));
+                } catch (DataException e) {
+                    logger.error("Problem couldn't be solved: " + e);
```

Comment:
2 things:
1) If a `DataException` is thrown is it a good idea to go on?
2) Did this kind of error logging work with casting exception object to string message?

Reply:
Look previous.